core/src/main/java/org/polypheny/db/catalog/impl/logical/RelationalCatalog.java

```diff
@@ -558,7 +557,7 @@ public void deleteConstraint( long constraintId ) {
             constraints.remove( logicalConstraint.id );
         }
         deleteKeyIfNoLongerUsed( logicalConstraint.keyId );
-        change();
+        change( "constraint", constraintId, null );
```

Comment:
Wouldn't it be better to use enums instead of strings here? It's the catalog, this cannot be extended by plugins, so we know exactly which events exist.

Reply:
Agreed, replaced with extensive enums collection covering various catalog operations.